cmd/buildctl/debug/info.go

```diff
@@ -0,0 +1,30 @@
+package debug
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	bccommon "github.com/moby/buildkit/cmd/buildctl/common"
+	"github.com/urfave/cli"
+)
+
+var InfoCommand = cli.Command{
+	Name:   "info",
+	Usage:  "display internal information",
+	Action: info,
+}
+
+func info(clicontext *cli.Context) error {
+	c, err := bccommon.ResolveClient(clicontext)
+	if err != nil {
+		return err
+	}
+	res, err := c.Info(bccommon.CommandContext(clicontext))
+	if err != nil {
+		return err
+	}
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', 0)
+	_, _ = fmt.Fprintf(w, "BuildKit:\t%s %s %s\n", res.BuildkitVersion.Package, res.BuildkitVersion.Version, res.BuildkitVersion.Revision)
```

Comment:
Can be another PR, but it would be nice to have JSON output too

Reply:
Is the one from https://github.com/moby/buildkit/pull/2725#issuecomment-1092751101 not enough?

```
    ...
    "buildkitVersion": {
      "Package": "github.com/moby/buildkit",
      "Version": "v0.10.0-43-gd2a757f8",
      "Revision": "d2a757f862957bf8a4cfb6f1bb990fa6dc33d153"
    }
  }
]
```

Reply:
Yes, currently enough.
We can revisit this when `InfoResponse` grows up and diverges from `WorkerRecord`.